airbyte-integrations/connectors/source-sentry/unit_tests/integration/test_events.py

```diff
@@ -0,0 +1,119 @@
+# Copyright (c) 2024 Airbyte, Inc., all rights reserved.
+
+# Import conftest helper
+import sys
+from datetime import datetime, timezone
+from pathlib import Path
+from unittest import TestCase
+
+import freezegun
+
+from airbyte_cdk.models import SyncMode
+from airbyte_cdk.test.catalog_builder import CatalogBuilder
+from airbyte_cdk.test.entrypoint_wrapper import read
+from airbyte_cdk.test.mock_http import HttpMocker
+
+# Import builders
+from integration.config import ConfigBuilder
+from integration.request_builder import SentryRequestBuilder
+from integration.response_builder import create_response, error_response
+
+
+sys.path.insert(0, str(Path(__file__).parent.parent))
+from conftest import get_source
+
+
+# Test constants
+_NOW = datetime.now(timezone.utc)
+_STREAM_NAME = "events"
+_ORGANIZATION = "test-org"
+_PROJECT = "test-project"
+_AUTH_TOKEN = "test_token_abc123"
+
+
+@freezegun.freeze_time(_NOW.isoformat())
+class TestEventsStream(TestCase):
+    """Comprehensive tests for events stream"""
+
+    def _config(self) -> dict:
+        """Helper to create config using builder"""
+        return ConfigBuilder().with_organization(_ORGANIZATION).with_project(_PROJECT).with_auth_token(_AUTH_TOKEN).build()
+
+    @HttpMocker()
+    def test_full_refresh_single_page(self, http_mocker: HttpMocker):
+        """
+        Test that connector correctly fetches one page of events.
+
+        This tests:
+        - Correct URL is called
+        - Auth header is set properly
+        - Response is parsed correctly
+        """
+        # ARRANGE
+        http_mocker.get(
+            SentryRequestBuilder.events_endpoint(_ORGANIZATION, _PROJECT, _AUTH_TOKEN).build(), create_response("events", has_next=False)
+        )
```

Comment:
If you look at the implementation of the `events` stream, we also include a query parameter for `full=true`, we should also be including this in the mock request we expect to validate we're passing that in properly.

I don't see that in the expected request unless I'm mistaken